docs/basic_usage.rst

```diff
@@ -40,3 +40,27 @@ NOTE: Remember to change the secret key of your application, and insure that no
 one is able to view it. The json web tokens are signed with the secret key, so
 if someone gets that, they can create arbitrary tokens, and in essence log in
 as any user.
+
+Partially protecting routes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There may be cases where you want to use one endpoint for both protected
+and unprotected data. In these situations, you can use the jwt_optional
+decorator. This will allow the view to be called whether or not an
+authorization header is sent in the request, although if there is a header
+that is expired, improperly formatted, or otherwise incorrect, an error
+will be returned.
+
+.. code-block:: python
+
+  @app.route('/partially-protected', methods=['GET'])
+  @jwt_optional
+  def partially_protected():
+      # If no JWT is sent in the request headers, get_jwt_identity()
+      # will return an empty dict
```

Comment:
This is actually a bug. I want ```get_jwt_identity()``` to return None, not an empty dict, if it isn't present. I'll get a new version that updates that pushed out today. Mind updating the comment here?